### DISCUSSION TITLE
Cannot see how much token available for staking or Unstaking

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#87] [General]
![Screenshot 2023-05-25 at 14 15 00](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/f01ae405-ea09-483b-87a8-ad204f7e2492)
this prompt should display the amount of asset available for staking.

For Unstaking same issue.

**Comment by TJ-BitCanna** [comment]
Check!

To be taken into account :)

**Comment by TJ-BitCanna** [comment]
Suggested issue:
- implement amount of tokens available to send, delegate, undelegate to be seen on the pop-ups

**Comment by TJ-BitCanna** [comment]
Tested and approved.
